snafu/run_snafu.py

```diff
@@ -73,6 +73,8 @@ def main():
             es['port'] = os.environ["es_port"]
             logger.info("Using elasticsearch server with port:" + es['port'])
     es_verify_cert = os.getenv("es_verify_cert", "true")
+    if es['port'] == 443:
```

Comment:
A few lines up, if this is set via-environment variable then it may be a string as opposed to an integer. The condition would fail in that case:
```
>>> 443 == '443'
False
```

Reply:
Ah ha! good point, it could be due to it being an env var.